scripts/rnnlm/choose_features.py

```diff
@@ -131,16 +141,30 @@ def read_unigram_probs(unigram_probs_file):
         if wordlist[idx] in SPECIAL_SYMBOLS:
             continue
         if p > 0.0:
-            entropy += math.log(p) * p;
+            entropy += math.log(p) * p
             entropy_total += p
     entropy /= -entropy_total
 
-    print("{0}\tunigram\t{1}\t{2}".format(num_features, entropy, args.unigram_scale))
+    rms = 0.0
+    for idx, p in enumerate(unigram_probs):
+        if wordlist[idx] in SPECIAL_SYMBOLS:
+            continue
+        if p > 0.0:
+            rms += (p - entropy) ** 2
+    rms = math.sqrt(rms / vocab_size)
+
+    print("{0}\tunigram\t{1}\t{2}".format(num_features, entropy, get_feature_scale(rms)))
```

Comment:
I think this could do with some more explanation at this point.  But maybe later- I may merge this as-is.